sympy/calculus/util.py

```diff
@@ -253,6 +253,83 @@ def elm_domain(expr, intrvl):
         return _domain
 
 
+def periodicity(f, symbol):
+    """Returns the period of a given function, if possible.
+
```

Comment:
Doesn't the 1st line of the docstring give sufficient information about the function ?